src/NamespaceGenerator.ts

```diff
@@ -452,8 +456,11 @@ class Generator {
                 ) + '";')
             );
 
-            this.moduleNamespace.addChildren(factoryDeclaration);
-            this.moduleNamespace.exports.push('export default factory;');
+            if (!Config.withoutFactory && factoryDeclaration) {
+                this.moduleNamespace.addChildren(factoryDeclaration);
+                this.moduleNamespace.exports.push('export default factory;');
+            }
+
```

Comment:
Looks good, but we need a default export for the Highcharts namespace when "withoutFactory".